lib/internal/errors.js

```diff
@@ -1308,8 +1308,12 @@ E('ERR_SERVER_NOT_RUNNING', 'Server is not running.', Error);
 E('ERR_SOCKET_ALREADY_BOUND', 'Socket is already bound', Error);
 E('ERR_SOCKET_BAD_BUFFER_SIZE',
   'Buffer size must be a positive integer', TypeError);
-E('ERR_SOCKET_BAD_PORT',
-  '%s should be >= 0 and < 65536. Received %s.', RangeError);
+E('ERR_SOCKET_BAD_PORT', function(name, port, allowZero = true) {
```

Comment:
use anonymous arrow function I think. Same to above.

Reply:
Yes, I see that the function of `ERR_TLS_CERT_ALTNAME_INVALID` does not use the arrow anonymous function, I am consistent with most of it